data/json/items/melee/spears_and_polearms.json

```
@@ -410,6 +410,38 @@
     "use_action": [ "HEAT_SOLID_ITEMS" ],
     "melee_damage": { "bash": 7, "stab": 24 }
   },
+  {
+    "type": "ITEM",
+    "id": "simple_pike_pipe",
+    "name": { "str": "basic pipe pike" },
+    "description": "5 pipes threaded together to form a very crude pike.  Heavy, slow, and all around cumbersome to carry and fight with.",
+    "weight": "5 kg",
+    "price_postapoc": "1 USD",
+    "color": "light_gray",
+    "looks_like": "spear_pipe",
+    "symbol": "/",
+    "material": [ "steel" ],
+    "techniques": [ "WBLOCK_1" ],
+    "volume": "4 L",
+    "longest_side": "3 meter",
+    "to_hit": { "grip": "solid", "length": "long", "surface": "point", "balance": "neutral" },
+    "flags": [ "DURABLE_MELEE", "POLEARM", "SPEAR", "REACH_ATTACK", "REACH3" ],
+    "weapon_category": [ "SPEARS" ],
+    "price": "120 USD",
+    "qualities": [ [ "COOK", 1 ] ],
+    "use_action": [ "HEAT_SOLID_ITEMS" ],
+    "melee_damage": { "bash": 8, "stab": 21 }
+  },
+  {
+    "type": "ITEM",
+    "id": "pike_pipe",
+    "copy-from": "simple_pike_pipe",
+    "name": { "str": "pipe pike" },
+    "description": "5 pipes welded together to form a very crude pike.  Heavy, slow, and all around cumbersome to carry and fight with.",
+    "weight": "3200 g",
+    "price_postapoc": "1 USD 50 cent",
+    "melee_damage": { "bash": 6, "stab": 23 }
```

```suggestion
    "melee_damage": { "bash": 8, "stab": 23 }
```
Why would the bash go down 🤔